submissions/Yasya23/holi-style-popup/index.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Menu</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <nav class="navigation">
+      <ul class="navigation__list">
+        <li class="navigation__item">
+          <a href="#">Mail</a>
+        </li>
+        <li class="navigation__item">
+          <a href="#">Images</a>
+        </li>
+        <li class="navigation__item">
+          <label for="popup">
+            <img
+              src="icons/popup-button.svg"
+              alt="Popup icon"
+              class="navigation__popup-icon"
+            />
+          </label>
+          <input type="checkbox" id="popup" class="popup__input" />
+          <div class="popup__container">
+            <ul class="popup__list">
+              <li class="popup__box-img">
+                <a href="#">
+                  <img
+                    src="icons/Collection.svg"
+                    alt="Collection icon"
```

Comment:
This is not a correct use of `alt` attribute, please check
https://www.w3.org/QA/Tips/altAttribute

Reply:
Thanks, got the point